ksql-execution/src/main/java/io/confluent/ksql/execution/expression/tree/StructExpression.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"; you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.execution.expression.tree;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.errorprone.annotations.Immutable;
+import io.confluent.ksql.parser.NodeLocation;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
+
+@Immutable
+public class StructExpression extends Expression {
```

Comment:
I find this name a bit vague, would `StructConstructorExpression` not be a more descriptive name?

Reply:
Changed to `CreateStructExpression`